modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-pvcs-to-store-default-os-images_{context}"]
+= Creating PersistentVolumeClaims to store default OS images
+
+Follow these steps to create a PersistentVolumeClaim (PVC), which you use to upload and save default OS images.
+
+.Prerequisites
+
+* You must have administrator privileges to upload and save default OS images.
+
+.Procedure
+
+. Select a bootable disk that you want to upload and save as a default OS image.
+
+. In the {product-title} Web Console, click *Storage > Persistent Volume Claims* in the sidebar menu. The *Persistent Volume Claim* page displays.
+
+. Click the *Create Persistent Volume Claim* button and select the *With Data upload form* option.
+
+. Complete the *Upload Data to Persistent Volume Claim* form to create a PVC that is used to upload and save default OS images.
+.. Click *Browse* and locate a bootable image to upload and save as a default OS image.
+.. Select the *Attach this disk to a VirtualMachine operating system* check box.
+.. Select the operating system of the bootable disk that you want to upload from the *Operating System* drop-down list.
+.. Select the storage class you want to use in the *Storage Class* drop-down list.
+.. Enter the size of the PVC you are creating in the *Size* fields.
```

Comment:
Is plural "fields" correct?

Reply:
It is, because in addition to entering the numerical size you can also enter the measurement (GiB etc.)